pcdsdevices/epics_motor.py

```diff
@@ -479,6 +492,8 @@ def Motor(prefix, **kwargs):
     """
     # Available motor types
     motor_types = (('MMS', IMS),
+                   ('CLZ', IMS),
```

Comment:
I believe that this is a duplicate commit?

https://github.com/pcdshub/pcdsdevices/blob/444f7a1a3a0ad978113ec0ed4c86a4bdec1f8ffd/pcdsdevices/epics_motor.py#L482

Think this was added in your last PR @silkenelson